packages/client/hmi-client/src/api/api.ts

```diff
@@ -20,7 +22,29 @@ API.interceptors.request.use(
 		return config;
 	},
 	(error) => {
-		console.error(error);
+		logger.error(error, { showToast: true, toastTitle: `Authentication Error` });
```

Comment:
I don't think we want to do this here, we should do like on the `App.vue` and redirect the user to the login page.

Also can we merge those two interceptors (from `App.vue` and here)? I don't understand why we should have two.